go/stats/prometheusbackend/prometheusbackend.go

```diff
@@ -67,8 +64,11 @@ func (be PromBackend) publishPrometheusMetric(name string, v expvar.Var) {
 		newMultiTimingsCollector(st, be.buildPromName(name))
 	case *stats.Histogram:
 		newHistogramCollector(st, be.buildPromName(name))
+	case *stats.String, stats.StringFunc, stats.StringMapFunc, *stats.Rates:
+		// silently ignore these types since they don't make sense to
```

Comment:
nit:

personal pet peeve of mine: Make this a proper sentence and use proper capitalization ("Prometheus'"):

> // Silently ignore these types since they don't make sense to
> // export to Prometheus' data model.

Reply:
Done